homeassistant/components/sensor/linky.py

```diff
@@ -75,17 +67,19 @@ def unit_of_measurement(self):
     @Throttle(SCAN_INTERVAL)
     def update(self):
         """Fetch new state data for the sensor."""
-        from pylinky.client import PyLinkyError
+        from pylinky.client import LinkyClient, PyLinkyError
+
+        client = LinkyClient(self._username, self._password)
```

Comment:
You probably need to close the session after `fetch_data()`.
See also [pyLinky main](https://github.com/Pirionfr/pyLinky/blob/master/pylinky/__main__.py#L24).
```
    finally:
        client.close_session()
```